applications/CoSimulationApplication/python_scripts/solver_wrappers/sdof/sdof_static_solver.py

```diff
@@ -0,0 +1,144 @@
+from __future__ import print_function, absolute_import, division  # makes these scripts backward compatible with python 2.6 and 2.7
+# Importing the base class
+import KratosMultiphysics
+from KratosMultiphysics.CoSimulationApplication.function_callback_utility import GenericCallFunction
+
+# Other imports
+import numpy as np
+import json
+import os
+
+class SDoFStaticSolver(object):
```

Comment:
I will talk to Andreas how to refactor this such that we don't have to duplicate code
=> refactor to have a common baseclass for both sdofs